lib/double_entry/transfer.rb

```diff
@@ -17,11 +17,10 @@ def code_max_length
       # @api private
       def transfer(amount, options = {})
         fail TransferIsNegative if amount < Money.zero
-        from = options[:from]
-        to = options[:to]
+        from_acct = options[:from]
```

Comment:
nice change! i think id prefer to go the full step and name them `from_account` and `to_account`